modules/shorewall/manifests/four/rule.pp

```diff
@@ -67,4 +67,12 @@
     ],
   }
 
+  shorewall::rule {
+'outgoing-mail':
+    source => 'all',
+    destination => 'net',
+    action  => 'SMTP(DROP)',
```

Comment:
@t-8ch in welche datei soll das genau? ich finde keine supernode.pp im shorewall kontext.